out-adapter-persistence-jpa/src/main/java/gohigher/application/entity/ApplicationProcessRepository.java

```diff
@@ -1,6 +1,12 @@
 package gohigher.application.entity;
 
+import java.util.Optional;
+
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface ApplicationProcessRepository extends JpaRepository<ApplicationProcessJpaEntity, Long> {
+
+	@Query("SELECT ap FROM ApplicationProcessJpaEntity ap WHERE ap.id = :id AND ap.application.id = :applicationId")
+	Optional<ApplicationProcessJpaEntity> findByIdAndApplicationId(Long id, Long applicationId);
```

Comment:
```suggestion
	@Query("SELECT ap FROM ApplicationProcessJpaEntity ap 
	WHERE ap.id = :id 
	AND ap.application.id = :applicationId")
	Optional<ApplicationProcessJpaEntity> findByIdAndApplicationId(Long id, Long applicationId);
```
쿼리의 가독성을 위해 줄바꿈을 하는건 어떨까요?